src/azul/project/hca/__init__.py

```diff
@@ -473,5 +474,64 @@ def portal_integrations_db(self) -> JSON:
                         "portal_url": "https://www.ebi.ac.uk/gxa/sc/experiments/E-ENAD-15/results/tsne"
                     },
                 ],
+            },
+            {
+                "portal_id": "2e05f611-16fb-4bf3-b860-aa500f0256de",
+                "portal_name": "Xena",
+                "portal_icon": "https://xenabrowser.net/03340e094d1f3edc51bc3d1a2a589b65.png",
+                "contact_email": "",
+                "organization_name": "UCSC",
+                "portal_description": "",
+                "integrations": [
+                    {
+                        "integration_id": integration_id,
+                        "integration_type": "get",
+                        "entity_type": "project",
+                        "title": title,
+                        "entity_ids": {
+                            "staging": [],
+                            "integration": [prod_uuid],
+                            "prod": [prod_uuid]
+                        },
+                        "portal_url": "https://singlecell.xenabrowser.net/datapages/?cohort=" + quote(title)
+                    } for integration_id, title, prod_uuid in (
```

Comment:
1) title should come last
2) rename prod_uuid to project_uuid
3) one tuple per line, and suppress warning

See diff